src/super_gradients/training/utils/collate_fn/ppyoloe_collate_fn.py

```diff
@@ -16,13 +16,19 @@ class PPYoloECollateFN(DetectionCollateFN):
     Collate function for PPYoloE training
     """
 
-    def __init__(self, random_resize_sizes: Union[List[int], None] = None, random_resize_modes: Union[List[int], None] = None):
+    def __init__(
+        self, random_resize_sizes: Union[List[int], None] = None, random_resize_modes: Union[List[int], None] = None, random_aspect_ratio: bool = False
+    ):
         """
-        :param random_resize_sizes: (rows, cols)
+        :param random_resize_sizes: List of sizes to randomly resize the image to. If None, will not resize.
+        :param random_resize_modes: List of interpolation modes to randomly resize the image to. If None, will not resize.
+        :param random_aspect_ratio: If True, will randomly choose both width and height from random_resize_sizes.
```

Comment:
This would be a bit more clear to me:
"If True, the height and width sizes will be drawn independently from random_resize_sizes".

Reply:
Updated docs in 664599da5e0148804dcd055ef68857b2ecfdeff3